test/tests/auxkernels/bounds/constant_bounds_fv.i

```diff
@@ -0,0 +1,118 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  xmin = 0
+  xmax = 1
+  nx = 10
+[]
+
+[Variables]
+  [u]
+    order = CONSTANT
+    family = MONOMIAL
+    fv = true
```

Comment:
```suggestion
  type = MooseVariableFVReal
```

Reply:
was kind of doing this on purpose to make sure we re resilient to other syntax

Reply:
ok that's fine